composer.json

```diff
@@ -39,7 +39,9 @@
     "require": {
         "php": "^5.5.0|^7.0",
         "payum/iso4217": "~1.0",
-        "guzzlehttp/guzzle": "~6.0",
+        "php-http/client-implementation": "^1.0",
+        "php-http/discovery": "^0.8",
+        "guzzlehttp/psr7": "^1.1",
```

Comment:
We are still using the psr7 implementation of guzzle. Not the http client. 
I could remove this dependency as well if you want.